.pending/improvements/modules/4821-types-errors-pa

```diff
@@ -0,0 +1,2 @@
+#4821 types/errors package added with support for stacktraces
```

Comment:
@ethanfrey do you think instead of the `modules` tag, we instead introduce a new tag `types` in `.clog.yaml`?

Reply:
Great idea. That or `sdk`, or `generic`

clog didn't let me use another tag. Feel free to push a commit to allow that and rename the clog, as I don't really understand how it works.

Reply:
Nevermind, I think I figured it out.
Please review `232a2f2` to make sure I didn't break anything

Reply:
Yup -- that's perfect. Thanks!